code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -274,6 +274,10 @@
 /obj/item/gun/magic/tentacle/shoot_with_empty_chamber(mob/living/user as mob|obj)
 	to_chat(user, span_warning("The [name] is not ready yet."))
 
+/obj/item/gun/magic/tentacle/attack_secondary(mob/living/victim, mob/living/user, params)
+	. = ..()
```

Comment:
```suggestion
	..()
```
. =  sets the return value, which is pointless if you immediately return something else